internal/clientcache/internal/db/schema.sql

```diff
@@ -13,6 +13,35 @@ create table if not exists user (
     check (length(address) > 0)
 );
 
+-- Contains the known resource types contained in the boundary client cache
+create table if not exists resource_type_enm(
+  string text not null primary key
+    constraint only_predefined_resource_types_allowed
+    check(string in ('unknown', 'target', 'session'))
+);
+
+insert into resource_type_enm (string)
+values
+  ('unknown'),
+  ('target'),
+  ('session');
+
+-- Contains refresh tokens for list requests sent by the client daemon to the
+-- boundary instance.
+create table if not exists refresh_token(
+  user_id text not null
+    references user(id)
+    on delete cascade,
+  resource_type text not null
+    references resource_type_enm(string)
+    constraint only_known_resource_types_allowed,
+  refresh_token text not null
+    check (length(refresh_token) > 0),
+  update_time timestamp not null default (strftime('%Y-%m-%d %H:%M:%f','now')),
```

Comment:
Do we want a trigger for these columns?  See https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/migrations/sqlite/02_version.up.sql